holidays/countries/barbados.py

```diff
@@ -0,0 +1,70 @@
+from holidays.calendars.gregorian import JAN, APR, JUL, AUG, NOV, MON
+from holidays.holiday_base import HolidayBase
+from holidays.holiday_groups import ChristianHolidays, InternationalHolidays
+
+
+class Barbados(HolidayBase, ChristianHolidays, InternationalHolidays):
+    """
+    https://en.wikipedia.org/wiki/Public_holidays_in_Barbados
+    https://www.timeanddate.com/holidays/barbados/
+    """
+
+    country = "BB"
+
+    special_holidays = {
+        # One off 50th Anniversary of CARICOM Holiday.
+        # See https://tinyurl.com/brbhol
+        2023: (JUL, 31, "50th Anniversary of CARICOM Holiday")
+    }
+
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate(self, year):
+        super()._populate(year)
+
+        # New Year's Day
+        self._add_new_years_day("New Year's Day")
+
+        # Errol Barrow Day
+        self._add_holiday("Errol Barrow Day", JAN, 21)
+
+        # Good Friday
+        self._add_good_friday("Good Friday")
+
+        # Easter Monday
+        self._add_easter_monday("Easter Monday")
+
+        # National Heroes Day
+        self._add_holiday("National Heroes Day", APR, 28)
+
+        # May Day
+        self._add_labor_day("May Day")
+
+        # Whit Monday
+        self._add_whit_monday("Whit Monday")
+
+        # Emancipation Day
+        self._add_holiday("Emancipation Day", AUG, 1)
+
+        # Kadootment Day
+        self._add_holiday("Kadooment Day", self._get_nth_weekday_of_month(1, MON, AUG))
```

Comment:
This method is no longer available, now we need to use `_get_nth_weekday_of_month()` from `holidays.calendars.gregorian` instead.

Reply:
Hmm that's weird, I can see it in HolidayBase and local test is passing.

Reply:
Right, I can see it's there in the master branch but not in this one. If I change it, local tests will fail - is that expected?

Reply:
Sorry, commented too soon. I got it.

Reply:
Or better, use new `_add_holiday_*` syntax, introduced in #1368:
```python
self._add_holiday_1st_mon_of_aug("Kadooment Day")
```

Reply:
Ah, I just pushed the changes with the other way. I can change it to this one if it is preferred.

Reply:
Yes, will be better to change (this holiday and all relevant holidays) to new syntax.

Reply:
Sure, just pushing the change now after tests.

Reply:
And done :)